src/loginpage/RegistrationForm.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import Input from '../components/Input';
+import Label from '../components/Label';
+import ColorInput from '../components/ColorInput';
+import SubmitButton from '../components/SubmitButton';
+
+const uid = require('uid');
+
+const StyledRegistrationForm = styled.form``;
+
+export default function RegistrationForm({
+  onNewUserGroup,
+  onLogin,
+  history,
+  userGroup
+}) {
+  function handleRegistration(event, onNewUserGroup, onLogin, history) {
```

Comment:
we had this before :) here again you dont need  `onNewUserGroup` `onLogin` and `history` as parameters and arguments